CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,65 @@
+
+* **Be respectful.** Not all of us will agree all the time, but disagreement is no excuse for poor behavior and poor manners. We might all experience some frustration now and then, but we cannot allow that frustration to turn into a personal attack. It's important to remember that a community where people feel uncomfortable or threatened is not a productive one. Members of the ESLint community should be respectful when dealing with other members as well as with people outside the ESLint community.
+
+* **Be careful in the words that you choose.** We are a community of professionals, and we conduct ourselves professionally. Be kind to others. Do not insult or put down other participants. Harassment and other exclusionary behavior aren't acceptable. This includes, but is not limited to:
+
+    * Violent threats or language directed against another person.
+    * Discriminatory jokes and language.
+    * Posting sexually explicit or violent material.
+    * Posting (or threatening to post) other people's personally identifying information ("doxing").
+    * Personal insults, especially those using racist or sexist terms.
+    * Unwelcome sexual attention.
+    * Advocating for, or encouraging, any of the above behavior.
+    * Repeated harassment of others. In general, if someone asks you to stop, then stop.
+
+* **When we disagree, try to understand why.** Disagreements, both social and technical, happen all the time and ESLint is no exception. It is important that we resolve disagreements and differing views constructively. Remember that we're different. The strength of the ESLintcommunity comes from inclusion of people from a wide range of backgrounds. Different people have different perspectives on issues. Being unable to understand why someone holds a viewpoint doesn't mean that they're wrong. Don't forget that it is human to err and blaming each other doesn't get us anywhere. Instead, focus on helping to resolve issues and learning from mistakes.
+
+
+## Reporting Guidelines
+
+If you believe someone is violating the code of conduct we ask that you report it to the jQuery Foundation by emailing [eslint-conduct@googlegroups.com](mailto:eslint-conduct@googlegroups.com). This is a private mailing list only viewable by project leads and cannot be accessed publicly except to send emails. All reports will be kept confidential. In some cases we may determine that a public statement will need to be made. If that's the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise.
+
+If you believe anyone is in physical danger, please notify appropriate law enforcement first.
```

Comment:
I'm wondering if this should be moved down? The following paragraph could be misconstrued to refer to the law enforcement report, rather than the report to the ESLint conduct inbox.